libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/DirectionsSession.kt

```diff
@@ -5,17 +5,17 @@ import com.mapbox.api.directions.v5.models.RouteOptions
 
 internal interface DirectionsSession {
 
-    fun getRoutes(): List<DirectionsRoute>
+    var routes: List<DirectionsRoute>
```

Comment:
shouldn't it be? you can keep `var` in implementation
```suggestion
    val routes: List<DirectionsRoute>
```

Reply:
Routes are settable from the outside, they need to be a `var`.